agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3728,7 +3728,7 @@ bool slave_thread::ap_manager_heartbeat_check()
     return true;
 }
 
-bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
+bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx, bool multicast)
```

Comment:
This should be done automatically - why do you need a parameter? You can check the message type and use multicast if the message type is one of the messages defined in Table 6-4.

Reply:
Good idea.